### DISCUSSION TITLE
Howto insert current date with a custom Date format?

### DISCUSSION THREAD

**Discussion by indeedwatson** [gsantner/markor#1458] [Q&A] [answered]
Hello, I'd like to have a different date format (specifically 2020.12.31 instead of 2020-12-31).

Searching around, it seems there used to be an option to customize the date format when you insert it, and I seem to remember that dialogue box, but when I add a date now it just pops up a calendar.

I went through the settings and can't find anything either.

**Comment by gsantner** [comment, marked as answer]
different date format for what? Use the attachment menu, it contains a date option with some presets or custom format.

**Reply by indeedwatson** [reply]
thanks, did not imagine it would be under attachments
